src/amo/utils/fxa.js

```diff
@@ -0,0 +1,5 @@
+/* @flow */
+export const MZA_LAUNCH_DATETIME: Date = new Date(
+  Date.UTC(2023, 10, 1, 19, 0, 0, 0),
+);
+export const isMzaBranding = (): boolean => new Date() > MZA_LAUNCH_DATETIME;
```

Comment:
should we compare against the (current)date in UTC as well?

Reply:
Good question! Do javascript needs to be explicitly converted to UTC before they're compared or does that one was created as UTC make it "just work"?

Reply:
the getTime() method tracks the milliseconds that has passed since jan 1 1970 UTC. So that should be enough. ```  new Date().getTime() >= MZA_LAUNCH_DATETIME.getTime();```

Reply:
Sorry, I should have replied earlier: I think this should work because `new Date(Date.UTC(...))` creates a local date based on the UTC one, and then we do the comparison against the current local date.